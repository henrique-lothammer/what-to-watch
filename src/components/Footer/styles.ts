import styled from 'styled-components'

export const Container = styled.div`
  padding: 10px 10px;
  position: absolute;
  bottom: 0;
  left: 0;
  text-align: center;
  width: 100%;

  span {
    color: red;
  }
`
