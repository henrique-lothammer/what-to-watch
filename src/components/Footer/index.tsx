import React, { ReactElement } from 'react'

import { Container } from './styles'

const Footer = (): ReactElement => {
  return (
    <Container data-testid='footer'>
      Made with <span>&#x2764;</span>, TMDB and React
    </Container>
  )
}

export default Footer
