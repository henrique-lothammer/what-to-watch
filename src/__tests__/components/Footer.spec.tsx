import React from 'react'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import Footer from 'components/Footer'

describe('Footer Component', () => {
  it('Should render', () => {
    const { container, getByTestId } = render(<Footer />)

    const element = getByTestId(/footer/i)
    expect(container.firstChild?.childNodes).toMatchSnapshot()
    expect(element).toBeInTheDocument()
  })
})
