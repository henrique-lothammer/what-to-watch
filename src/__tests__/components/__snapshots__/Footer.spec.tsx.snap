// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Should render 1`] = `
NodeList [
  Made with ,
  <span>
    ‚ù§
  </span>,
  , TMDB and React,
]
`;
